nim/Dockerfile

```diff
@@ -1,16 +1,16 @@
 FROM nimlang/nim:1.6.4-alpine
 
-WORKDIR /usr/src/app
-
-COPY . ./
+RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/testing >> /etc/apk/repositories
```

Comment:
are you confident enough to run a **production** api with `testing`  packages ?

Reply:
deleted